src/draggable-grid.tsx

```diff
@@ -20,14 +20,15 @@ interface IBaseItemType {
   key:string;
 }
 
-export interface IDraggableGridProps<DataType extends IBaseItemType>{
+export interface IDraggableGridProps<DataType extends IBaseItemType> {
+  dragEnabled:boolean;
```

Comment:
is this should be `dragEnabled?:boolean;`?

Reply:
We can name it dragDisabled and set default value to false.